src/src/WebServer/ConfigPage.cpp

```diff
@@ -259,6 +262,15 @@ void handle_config() {
 
   addFormSeparator(2);
 
+  #if FEATURE_ALTERNATIVE_CDN_URL
+  addFormSubHeader(F("CDN Override"));
+
+  addFormTextBox(F("CDN Override URL"), F("alturl"), get_CDN_url_override(), 255);
+  addFormNote(F("Leave empty for default cdn.jdelivr.net url"));
```

Comment:
Maybe we can still use the compile-time defined strings cpp file to keep the cdn.jsdelivr.net.
This way a user can already define one in their Custom.h file and you can show the hostname of that default here in the note.

Reply:
When having a `Custom.h` defined CDN url, this entire feature is disabled. The default url I'll append in the UI 👍.